src/gtfs.rs

```diff
@@ -53,12 +53,89 @@ impl From<Agency> for objects::Company {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+struct Stop {
+    #[serde(rename = "stop_id")]
+    id: String,
+    #[serde(rename = "stop_code")]
+    code: Option<String>,
+    #[serde(rename = "stop_name")]
+    name: String,
+    #[serde(default, rename = "stop_desc")]
+    desc: String,
+    #[serde(rename = "stop_lon")]
+    lon: f64,
+    #[serde(rename = "stop_lat")]
+    lat: f64,
+    #[serde(rename = "zone_id")]
+    fare_zone_id: Option<String>,
+    #[serde(rename = "stop_url")]
+    url: Option<String>,
+    #[serde(default)]
+    location_type: i32,
+    parent_station: Option<String>,
+    #[serde(rename = "stop_timezone")]
+    timezone: Option<String>,
+    #[serde(default, rename = "wheelchair_boarding")]
```

Comment:
do you really need to rename `wheelchair_boarding` to `wheelchair_boarding`?

Reply:
you're right ! fixed